db/migrate/20171023170841_ensure_cloud_managers.rb

```diff
@@ -0,0 +1,47 @@
+class EnsureCloudManagers < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base; end
+  class BaseManager < ExtManagementSystem; end
+  class Settings < ActiveRecord::Base; end
+
+  class ManageIQ
+    class Providers
+      class CloudManager < BaseManager
```

Comment:
This should be modifying the stub model created above.

Reply:
@Fryguy Sorry, I don't follow.